Sources/PT.PM.AntlrUtils/AntlrBaseHandler.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using System.Text;
+using Antlr4.Runtime.Atn;
+using PT.PM.Common;
+using PT.PM.Common.Files;
+
+namespace PT.PM.AntlrUtils
+{
+    public abstract class AntlrBaseHandler : ILanguageHandler
+    {
+        protected static Dictionary<Language, ATN> lexerAtns = new Dictionary<Language, ATN>();
+
+        protected static Dictionary<Language, ATN> parserAtns = new Dictionary<Language, ATN>();
```

Comment:
`lexerAtns` and `parserAtns` should be moved to respective Lexer and Parser classes.

Reply:
maybe we should left only one collection in base class, with name atns?

Reply:
Ok, it's also possible.